config.js.template

```diff
@@ -1,10 +1,10 @@
 //globals: equal, responseText, statement, ok, deepEqual, QUnit, module, asyncTest, Util, start, golfStatements, console
 /*jslint bitwise: true, browser: true, plusplus: true, maxerr: 50, indent: 4 */
 function Config() {
-	"use strict";
+    "use strict";
 }
 Config.endpoint = "<TCAPI end point (ending with slash)>";
 Config.authUser = "<account id>";
 Config.authPassword = "<password>";
-Config.actor = { "mbox":["<learner email>"], "name":["<Learner Name>"] };
+Config.actor = { "mbox":"<learner email>", "name":"<Learner Name>" };
```

Comment:
Yeh, I definitely changed it in the launcher early on (so it would have been tested a lot during further development). Further, I just launched this version of the prototypes using ```Config.actor = { "mbox":"andrew.downes@tincanapi.com", "name":"Andrew Downes" };``` (which was already set up in my local copy) and it works.